projects/fundamental-ngx/src/lib/date-picker/date-picker.component.html

```diff
@@ -0,0 +1,16 @@
+<div class="fd-date-picker">
+    <div class="fd-popover">
+        <div class="fd-popover__control">
+            <div class="fd-input-group fd-input-group--after">
+                <input #datePicker type="text" [value]="inputFieldDate" placeholder="Pick a date" (change)="getInputValue(datePicker.value)" >
```

Comment:
Maybe the placeholder should display the expected format (mm/dd/yyyy)?

Reply:
I followed the Fundamental UI implementation

Reply:
use mm/dd/yyyy for placeholder